### PR TITLE
pg_ctl: add page

### PR DESCRIPTION
**Pull request by heimann**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by sbrl** [comment]
Thanks, @heimann! 😺

File: pages/common/pg_ctl.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`pg_ctl -D {{data_directory}} -l {{log_file_name}}`
+
+- Initialize a PostgreSQL database cluster:
+
+`pg_ctl -D {{data_directory}} init`
+
+- Stop a PostgreSQL server:
+
+`pg_ctl -D {{data_directory}} stop`
```
Comment (agnivade) [inline pages/common/pg_ctl.md]: Don't we need to be root for the start/stop operations ?

Comment (heimann) [reply pages/common/pg_ctl.md]: No, I think it's best practice to not run those as root. If you initialize the db and root owns the data directory then yes you need to be root to stop it, but the recommended way is to actually create a separate 'postgres' user and run it as that user:
```su postgres -c 'pg_ctl start -D /usr/local/pgsql/data -l serverlog'```
as per: https://www.postgresql.org/docs/9.1/static/server-start.html

Comment (agnivade) [reply pages/common/pg_ctl.md]: Right, thanks.

**Pull request merged**